conda/core/subdir_data.py

```diff
@@ -169,16 +167,6 @@ def __init__(self, channel, repodata_fn=REPODATA_FN):
         self._loaded = False
         self._key_mgr = None
 
-    @property
-    def _repo(self) -> RepoInterface:
-        # XXX one for `current_repodata.json` and one for `repodata.json`
-        return CondaRepoInterface(
-            self.url_w_credentials,
-            self.repodata_fn,
-            cache_path_json=self.cache_path_json,
-            cache_path_state=self.cache_path_state,
-        )
```

Comment:
we did wind up using this twice, giving us a single place to update when we need to pass different arguments for different parameters to CondaRepoInterface as might be needed for jlap. Should we have a test for this method